code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -83,6 +89,20 @@
 		add_blood(blood_color)
 	return
 
+/obj/item/weapon/melee/telebaton/proc/stun(mob/living/target, mob/living/user)
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(H.check_shields(src, 0, "[user]'s [name]"))
+			return FALSE
+	user.visible_message("<span class='danger'>[user] knocks down [target] with [src]!</span>",\
+							"<span class='danger'>You knock down [target] with [src]!</span>")
+	// Visuals and sound
+	playsound(target, 'sound/weapons/Genhit.ogg', 75, TRUE, 7)
+	log_interact(user, target, "[key_name(user)] stunned [key_name(target)] with [src]")
```

Comment:
not `log_attack`?

Reply:
log_attack doesn't actually log it to VV attack logs, and as it's not harmful I feel this would be more appropriate.

Reply:
sounds good